utils/conversions.ts

```diff
@@ -102,13 +102,11 @@ export const toModelBonusActivation = (
 export const toApiBonusActivation = (
   domainObj: BonusActivationWithFamilyUID
 ): Either<t.Errors, ApiBonusActivation> => {
-  const snakeCasedUntypedObj = renameObjectKeys(domainObj, k =>
-    camelCaseToSnakeCase(k)
-  );
-  return ApiBonusActivation.decode({
-    ...snakeCasedUntypedObj,
-    family_members: undefined
-  });
+  const {
+    family_members,
```

Comment:
why are we removing family_members here?

Reply:
it's a typo